src/components/Button.tsx

```diff
@@ -0,0 +1,38 @@
+import cx from 'classnames';
+
+type ButtonProps = {
+  children: React.ReactNode | string;
+  onClick?: () => void;
+  disabled?: boolean;
+  ariaLabel?: string;
+  className?: string;
+  variant?: 'primary' | 'secondary';
+  type?: 'button' | 'submit' | 'reset';
+};
+
+export const Button = ({
+  children,
+  onClick,
+  disabled,
+  ariaLabel,
+  className,
+  variant = 'primary',
+  type = 'button',
+}: ButtonProps) => (
+  <button
+    onClick={onClick}
+    disabled={disabled}
+    aria-label={ariaLabel}
+    type={type}
+    className={cx(
+      'min-w-[10em] rounded-md bg-button p-2 text-white',
```

Comment:
why do we use ems? That sounds like another level of variability, which is always hard to debug. What about either using rems or not setting min-width at all. 

I do imagine this as a component library that has JUST decided that small words like "Done" or "Ok" will have bigger padding on the sides than "Please confirm you are accepting this" or whatever long text

Reply:
Done. ✅